tests/configMergeTest.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+/**
+* @file
+*  Test config-merge, that merges configuration changes from one site to another.
+*/
+
+namespace Unish;
+
+/**
+ *  @group slow
+ *  @group commands
+ */
+class configMergeTest extends CommandUnishTestCase {
```

Comment:
Yeah, it's already tested.  See "Part 1" (git), "Part 2" (format-patch) and "Part 3" (rsync) in the tests.  I deliberately made non-isolated tests for the different modes, because it's supposed to be a feature of config-merge that you can keep running one merge after another, so if one part breaks the next one, then that's a bug.  (Except format-patch and rsync do break git, if you do one of those first -- that's a limitation, not a bug, so we test git first.)